object-storage-aws/src/main/java/io/micronaut/objectstorage/aws/AwsS3Operations.java

```diff
@@ -17,11 +17,21 @@
 
 import io.micronaut.context.annotation.EachBean;
 import io.micronaut.context.annotation.Parameter;
-import io.micronaut.objectstorage.*;
```

Comment:
I use IntelliJ import optimization. I wonder whether we should persist this via `.editorconfig` or similar. Otherwise, IDEA may want to revert your change the next time I change the file.

Reply:
You can configure the number of imports allowed before grouping with * in IntelliJ settings Editor->Code Style->Java->Imports - I changed the values to 100 and checked "Use single class import"